docs/docs/index.mdx

```diff
@@ -7,6 +7,15 @@ title: Introduction to Rasa Action Servers
 A Rasa action server runs [custom actions](https://rasa.com/docs/rasa/next/custom-actions) for a Rasa Open Source
 conversational assistant.
 
+Rasa will call an endpoint you can specify, when a custom action is predicted. This endpoint should be a webserver that reacts to this call, runs the code and optionally returns information to modify the dialogue state.
```

Comment:
I think this location is fine, but I'd definitely also add it [here](https://rasa.com/docs/rasa/custom-actions). What do you think? Can we also describe how to configure things like SSL or headers please?

Reply:
I also think it's fine to explain how to connect to an action server in the Rasa Open Source docs.

My understanding is that the Action Server docs in the Rasa Open Source doc is meant to be an overview of what an action server does and users should refer to the docs in the Rasa SDK if they want to learn more. So I think we should describe other configuration e.g. SSL, headers in the Rasa SDK docs. @ArjaanBuijk what do you think?

@wochinge Is this the [PR](https://github.com/RasaHQ/rasa/pull/9118) you are referring to describe the SSL config?  For the headers config,  what's the PR for it?

Reply:
> I also think it's fine to explain how to connect to an action server in the Rasa Open Source docs.

My understanding is that the Action Server docs in the Rasa Open Source doc is meant to be an overview of what an action server does and users should refer to the docs in the Rasa SDK if they want to learn more.

True. Makes sense 👍🏻  Good point! 

>  Is this the PR you are referring to describe the SSL config?

> For the headers config, what's the PR for it?

There is none (it's been there for ages). You can see from [this object ](https://github.com/RasaHQ/rasa/blob/f58acdea8408563c428282d3159b11d3973acacd/rasa/utils/endpoints.py#L79)what's possible to configure.

Reply:
This is a configuration you do with Rasa Open Source, so I feel it should be mentioned there. Or at least, it should provide the link to this sdk doc section how to configure the endpoint.